// Copyright 2018 Xcalar, Inc. All rights reserved.
//
// No use, or distribution, of this source code is permitted in any form or
// means without a valid, written license agreement with Xcalar, Inc.
// Please refer to the included "COPYING" file for terms and conditions
// regarding the use and redistribution of this software.
//

syntax = "proto3";

import "google/protobuf/any.proto";

message ServiceRequest {
    string serviceName = 1;
    string methodName = 2;
    google.protobuf.Any body = 3;
};

message ServiceResponse {
    google.protobuf.Any body = 1;
};
